src/crate/crash/command.py

```diff
@@ -240,15 +248,19 @@ def __init__(self,
         }
         self.commands.update(built_in_commands)
         self.logger = ColorPrinter(is_tty)
+
+        self.output_writer = output_writer or \
+            OutputWriter(PrintWrapper(), is_tty)
```

Comment:
I think we should add a `setup.cfg` `pycodestyle` entry and raise the line-length to something like 88  (while still aiming for 80 in most cases)

Breaks like this imho hurt readability more than they improve it. See also https://github.com/ambv/black#line-length

Reply:
It's better than it was before, though, I think. but I agree, that it makes sense to put it into a single line for readability.